content/docs/handling-events.md

```diff
@@ -1,72 +1,72 @@
 ---
 id: handling-events
-title: Handling Events
+title: Obsługa zdarzeń
 permalink: docs/handling-events.html
 prev: state-and-lifecycle.html
 next: conditional-rendering.html
 redirect_from:
   - "docs/events-ko-KR.html"
 ---
 
-Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+Obsługa zdarzeń w Reakcie przypomina tę z drzewa DOM. Istnieje jednak kilka różnic w składni:
 
-* React events are named using camelCase, rather than lowercase.
-* With JSX you pass a function as the event handler, rather than a string.
+* Zdarzenia reactowe pisane są camelCasem, a nie małymi literami.
```

Comment:
Wcześniej pisane było reakt przez 'k'. Tutaj piszemy przez 'c'. Wypadałoby się dogadać w tej kwestii. Osobiście wolałbym nie spolszczać tej nazwy (w końcu jest to nazwa własna biblioteki).

Reply:
To jest normalna odmiana słowa "React":
- Narzędnik: (z) Reactem,
- Miejscownik: (o) Reakcie
Zajrzyj do Issues, tam jest to wytłumaczone i poparte przykładem z poradnik PWN.

Reply:
@jakubdrozdek dzięki za wyjaśnienie. 👍